src/lib/system-info.ts

```diff
@@ -0,0 +1,114 @@
+import * as systeminformation from 'systeminformation';
+import * as osUtils from 'os-utils';
+import { fs, child_process } from 'mz';
+
+export function getCpuUsage() {
+	return new Promise((resolve) => {
+		osUtils.cpuUsage((percent) => {
+			resolve(Math.round(percent * 100));
+		});
+	});
+}
+
+const blockDeviceRegex = /(\/dev\/.*)p\d+/;
```

Comment:
What's the `p`?

Reply:
Matches against the partition

Reply:
@CameronDiver that's sd/emmc specific, it won't work for nucs or other devices that don't use that format

Reply:
It will work for an nvme nuc, will do further testing.

Reply:
Could you add a comment about what we expect to match - it could help a lot later on when looking back at this code